server/pulp/server/db/connection.py

```diff
@@ -163,6 +174,14 @@ def _connect_to_one_of_seeds(connection_kwargs, seeds_list, db_name):
             _logger.info(msg % {'url': seed, 'e': str(e)})
 
 
+def _get_mongo_version():
+    db_version = _CONNECTION.server_info()['version']
+    _logger.info(db_version)
+    _logger.info(_("Mongo database for connection is version %s") % db_version)
```

Comment:
I'm being picky, but mongo only manages our database, and that database is not versioned (at least not in this way). You're logging the version of mongo itself, so I'd change the language to something like "Connected to MongoDB version..."